src/qibolab/platforms/abstract.py

```diff
@@ -482,7 +426,10 @@ def create_RX_drag_pulse(self, qubit, start, relative_phase=0, beta=None):
         if beta != None:
             qd_shape = "Drag(5," + str(beta) + ")"
 
-        qd_channel = self.qubits[qubit].drive.name
+        if self.qubits[qubit].drive:
+            qd_channel = self.qubits[qubit].drive.name
+        else:
+            qd_channel = self.settings["qubit_channel_map"][qubit][1]
```

Comment:
This pattern is repeated 7 times: make a method for this look up :)
```py
def qd_channel(self, qubit):
    if self.qubits[qubit].drive:
        return self.qubits[qubit].drive.name
    return self.settings["qubit_channel_map"][qubit][1]
```

Reply:
Done. Just mentioning that this is temporary and ideally we should have one way of obtaining the channels. It is just like this now because the `MutliqubitPlatform` is still using the `qubit_channel_map` dictionary.